app/src/lib/git/stash.ts

```diff
@@ -39,17 +30,16 @@ export async function getDesktopStashEntries(
 
   try {
     result = await git(
-      ['log', '-g', 'refs/stash', `--pretty=${prettyFormat}`],
+      ['log', '-g', `--pretty=${prettyFormat}`, 'refs/stash'],
       repository.path,
       'getStashEntries'
     )
   } catch (err) {
     if (err instanceof GitError) {
-      if (
-        !expectedErrorMessages.some(
-          message => err.message.indexOf(message) !== -1
-        )
-      ) {
+      const shouldThrow = !expectedErrorMessages.some(
```

Comment:
As I said in https://github.com/desktop/desktop/issues/7254#issuecomment-482063241 I believe this code could be removed entirely if we were just to add `128` to the list of expected exit codes. I will open a PR to illustrate what I mean.

Reply:
Pr opened: https://github.com/desktop/desktop/pull/7268

Reply:
Merged